Adafruit_PM25AQI_UART_PMS5003.cpp

```diff
@@ -0,0 +1,3 @@
+// #include "Adafruit_PM25AQI_UART_PMS5003.h"
+
+// Adafruit_PM25AQI_UART_PMS5003::Adafruit_PM25AQI_UART_PMS5003() {}
```

Comment:
Probably remove file, unlikely to ever need a body

Reply:
Sure, remove it